### PR TITLE
PAI submitter for XGBoost training

### PR DESCRIPTION
**Pull request by lhw362950217**
Related to #2656

### CODE REVIEW

File: python/runtime/pai/submitter.py [hunk 1]
```diff
@@ -41,6 +41,26 @@
...
+def get_requirement(model_name):
+    """Get required python package according to estimator name
+
+    Args:
+        estimator: name of the model
+
+    Returns:
+        A string with multilines, each line is a requirement
+    """
+    if model_name.lower() == "xgboost":
```
Comment (sneaxiy) [inline python/runtime/pai/submitter.py]: How about `model_name.lower().startswith("xgboost")`? The `model_name` may be in the form of `xgboost.gbtree`.

File: python/runtime/pai/submitter.py [hunk 2]
```diff
@@ -381,7 +406,11 @@ def submit_pai_tf_train(datasource, estimator_string, select,
     params = dict(locals())
     del params["train_params"]
     params.update(train_params)
-    params["entry_type"] = "train"
+
+    if estimator_string.lower() == "xgboost":
```
Comment (sneaxiy) [inline python/runtime/pai/submitter.py]: Same above.

File: python/runtime/pai/xgboost/train.py [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+                   is_pai=True,
+                   pai_train_table=pai_table,
+                   pai_validate_table=pai_val_table,
+                   oss_model_dir=oss_model_dir,
+                   transform_fn=transform_fn,
+                   feature_column_code=feature_columns_code,
+                   model_repo_image=model_repo_image,
+                   original_sql=original_sql)
+    else:
+        local_train(datasource=datasource,
```
Comment (sneaxiy) [inline python/runtime/pai/xgboost/train.py:92]: Where is the definition of `local_train`?

Comment (lhw362950217) [reply python/runtime/pai/xgboost/train.py:92]: It is an alias : from runtime.xgboost.train import train as local_train

**Review by sneaxiy — APPROVED** [review]
LGTM++

**Pull request merged**